builds/e2e/e2e.yaml

```diff
@@ -1,38 +1,50 @@
-# End-to-end tests should run after new docker images and edgelet packages are
-# built, but 'build completion triggers' are not yet supported in YAML. Set to
-# 'none' here and override in the online editor.
 trigger: none
 pr: none
 
-jobs:
-
-################################################################################	
-  - job: linux_arm32v7
-################################################################################	
-    displayName: Linux arm32v7
-
-    pool:
-      name: $(pool.name)
-      demands: rpi3-e2e-tests
+resources:
```

Comment:
Just to share, currently there is a problem the e2e test is triggered by image build, but not edgelet build or both.  To solve this problem, should we use multi-stage build? https://docs.microsoft.com/en-us/azure/devops/pipelines/process/stages?view=azure-devops&tabs=yaml

Reply:
@philipktlin This build has two build completion triggers: image build _and_ packages build. It runs twice for every commit in master.

Reply:
Ideally we should just run once for each commit with these 2 artifacts from image build and edgelet build. right?

Reply:
Yes that's what I was hoping to achieve in this draft PR. Although now I'm thinking it might still run twice per commit because build images and edgelet packages seem to both run on every commit, not just on commits that impact the components they build. We may need to improve build images and edgelet packages to reduce the number of times we run this pipeline.